### ISSUE TITLE
Cython: cannot get relative cimport to work

### ISSUE THREAD

**Issue by williamhCode** [cython/cython#4860]
I have an issue that I have not figured out for a long time, and I already tried asking the google groups and stack overflow with no luck. I would be grateful if someone could find a solution to my issue. It may potentially be a bug too. Here is the Stack Overflow link - [Cython: cannot get relative cimport to work](https://stackoverflow.com/questions/72654024/cython-cannot-get-relative-cimport-to-work).

#### A Quick Overview on the Issue
Basically, I cannot get relative import between two pyx/pxd files to work, no matter what I've tried. I've seen old answers, tried them, but they don't work. Go to the link for more info.

**Comment by matusvalo** [comment]
@scoder @da-woods should we close this issue as duplicate to #3442? Can we close them by merging #4552?

**Comment by da-woods** [comment]
I think it is a duplicate as you say. Thanks
